dockerfile_scripts/build_nccl.sh

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/env bash
 
-set -e
+set -x
```

Comment:
Did you want to keep this as "-x" instead of "-e" ?

Reply:
I preferred -x to see what went wrong more but can put it back if it matters to you.